packages/documentation-v7/src/stories/components/select/select.stories.ts

```diff
@@ -0,0 +1,328 @@
+import type { Args, Meta, StoryContext, StoryObj } from '@storybook/web-components';
+import { html, unsafeStatic } from 'lit/static-html.js';
+import { BADGE } from '../../../../.storybook/constants';
+import { LitElement } from 'lit';
+import { useArgs } from '@storybook/preview-api';
```

Comment:
Import node modules first (group them if they come from the same module) and custom imports second.

```suggestion
import { LitElement } from 'lit';
import { html, unsafeStatic } from 'lit/static-html.js';
import { useArgs } from '@storybook/preview-api';
import { BADGE } from '../../../../.storybook/constants';
```